lighthouse-core/gather/gatherers/dobetterweb/response-compression.js

```diff
@@ -14,28 +14,41 @@ const Gatherer = require('../gatherer');
 const gzip = require('zlib').gzip;
 
 const compressionTypes = ['gzip', 'br', 'deflate'];
+const mediaTypes = ['mp3', 'wav', 'wma', 'webm', 'jpg', 'gif', 'png', 'webp',
+  'svg', 'mkv', 'flv', 'gif', 'avi', 'mov', 'mpg', '3gp', 'mpeg', 'avi', 'wmv'];
+
 const CHROME_EXTENSION_PROTOCOL = 'chrome-extension:';
 
 class ResponseCompression extends Gatherer {
   /**
-   * @param {!NetworkRecords} networkRecords
-   * @return {!Array<{url: string, isBase64DataUri: boolean, mimeType: string, resourceSize: number}>}
-   */
+     * @param {!NetworkRecords} networkRecords
+     * @return {!Array<{url: string, isBase64DataUri: boolean, mimeType: string, resourceSize: number}>}
+     */
   static filterUnoptimizedResponses(networkRecords) {
     const unoptimizedResponses = [];
 
     networkRecords.forEach(record => {
       const isTextBasedResource = record.resourceType() && record.resourceType().isTextType();
+      const url = record.url.toLowerCase();
```

Comment:
`const isMediaResource = (record.resourceType() === WebInspector.resourceTypes.Media || record.resourceType() === WebInspector.resourceTypes.Image;` is probably a better approach.

Although I think we want svg's to be compressed at they are text based and not binary based.

Reply:
in this case we're already being fooled by what Chrome thinks the resource type is unfortunately, let's try to add a mime type sanity check instead :)